config/initializers/devise.rb

```diff
@@ -9,11 +9,6 @@
   env["omniauth.strategy"].options[:scope] = "user:email"
 end
 
-FACEBOOK_OMNIAUTH_SETUP = lambda do |env|
-  env["omniauth.strategy"].options[:client_id] = SiteConfig.facebook_key
-  env["omniauth.strategy"].options[:client_secret] = SiteConfig.facebook_secret
```

Comment:
Adding the config this way worked for me locally:

```
env["omniauth.strategy"].options[:token_params][:parse] = 'json'
```

Although I'm not 100% sure if it makes a difference since it worked locally for me with or without it

Reply:
How did you verify that the syntax for the parse setting is correct? I couldn't find a way to inspect that setting programmatically during runtime, which made me a bit nervous!

Reply:
Ok: playing around and setting token_parse to `text` results in an error (in fact the one in product), so I'm confident this syntax is working!

Reply:
> Ok: playing around and setting token_parse to text results in an error (in fact the one in product), so I'm confident this syntax is working!

Oh nice smart move to test it that way! I hadn't verified tbh, it was an eyeball config from my end but good to hear it's being set correctly then :)